astro/src/content/docs/customize/email-and-messages/deprecated/index.mdx

```diff
@@ -4,6 +4,7 @@ description: Learn how to complete Two Factor authentication for a User
 section: customize
 subcategory: email and messages
 tertcategory: deprecated
+topOfNav: true
```

Comment:
This page is labeled as an index page and should be an overview, but it is for Two Factor Auth.  I don't think the topOfNav should apply here and could be renamed.  Would need a default page then.